common/views/components/ModalMoreFilters/ModalMoreFilters.tsx

```diff
@@ -129,6 +137,78 @@ const CheckboxFilter = ({ f, changeHandler, form }: CheckboxFilterProps) => {
   );
 };
 
+type DateRangeFilterProps = {
+  f: DateRangeFilterType;
+  changeHandler: () => void;
+  form?: string;
+  newStyle?: boolean;
+};
+
+const DateRangeFilter = ({ f, changeHandler, form }: DateRangeFilterProps) => {
+  const [from, setFrom] = useControlledState(f.from.value);
+  const [to, setTo] = useControlledState(f.to.value);
+
+  return (
+    <fieldset name={f.label} form={form}>
+      <Space as="span" h={{ size: 'm', properties: ['margin-right'] }}>
+        <NumberInput
+          name={f.from.id}
+          label="From"
+          min="0"
+          max="9999"
+          placeholder="Year"
+          value={from || ''}
+          onChange={event => {
+            const val = `${event.currentTarget.value}`;
+            setFrom(val);
+            if (val.match(dateRegex)) {
+              changeHandler();
+            }
+          }}
+          form={form}
+        />
+      </Space>
+      <NumberInput
+        name={f.to.id}
+        label="to"
+        min="0"
+        max="9999"
+        placeholder="Year"
+        value={to || ''}
+        onChange={event => {
+          const val = `${event.currentTarget.value}`;
+          setTo(val);
+          if (val.match(dateRegex)) {
+            changeHandler();
+          }
+        }}
+        form={form}
+      />
+    </fieldset>
+  );
+};
+
+const PaletteColorPicker = dynamic(
```

Comment:
For my own knowledge; why does this needs to be a dynamic import? (and should we leave a comment to explain it to future devs too)

Reply:
I've made the change